lib/prometheus/que_stats.rb

```diff
@@ -136,7 +136,7 @@ def call(type = nil)
       protected
 
       def type_hash_and_stats_count(type)
-        type_hash = type ? { type: type } : {}
+        type_hash = { type: type }
```

Comment:
Not sending an empty hash, same as https://github.com/3scale/zync/pull/283/files#diff-f903aea79c99fca32c1fafbb2b9e0cd134020edf41088b78f090b2a00faf7978R5